packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -188,7 +188,6 @@ class CurlyComponentManager {
     aliasIdToElementId(args, props);
 
     props.parentView = parentView;
-    props.renderer = parentView.renderer;
```

Comment:
Ya, I can fix. Might be easier to put it back for now and I'll fix with a follow up PR later this afternoon...